lib/hushed/documents/response/inventory_summary_document.rb

```diff
@@ -0,0 +1,52 @@
+module Hushed
+  module Documents
+    module Response
+      class InventorySummaryDocument
+
+        def initialize(options = {})
+          @io = options[:io]
+          @document = Nokogiri::XML::Document.parse(@io)
+        end
+
+        def client_id
+          @client_id ||= inventory_summary['ClientId']
+        end
+
+        def message_date
+          @message_date ||= Time.parse(inventory_summary['MessageDate'])
+        end
+
+        def business_unit
+          @business_unit ||= inventory_summary['BusinessUnit']
+        end
+
+        def warehouse
+          @warehouse ||= inventory_summary['Warehouse']
+        end
+
+        def item_stat_hash(item)
```

Comment:
i'm probably picky but 'stat' does not resonate anything for me.  `quantity_by_status` maybe?  don't have a better idea for now